api/python/v2/observation.md

```diff
@@ -7,6 +7,1835 @@ grand_parent: API - Query data programmatically
+            },
+            {
+              "facetId": "1151455814",
+              "observations": [
+                {
+                  "date": "2023",
+                  "value": 40097761
+                }
+              ],
+              "obsCount": 1,
+              "earliestDate": "2023",
+              "latestDate": "2023"
+            },
+            {
+              "facetId": "4181918134",
+              "observations": [
+                {
+                  "date": "2023",
+                  "value": 40097761
+                }
+              ],
+              "obsCount": 1,
+              "earliestDate": "2023",
+              "latestDate": "2023"
+            },
+            {
+              "facetId": "1216205004",
+              "observations": [
+                {
+                  "date": "2021",
+                  "value": 36991981
+                }
+              ],
+              "obsCount": 1,
+              "earliestDate": "2021",
+              "latestDate": "2021"
+            }
+          ]
+        }
+      }
+    }
+  },
+  "facets": {
+    "3981252704": {
+      "importName": "WorldDevelopmentIndicators",
+      "provenanceUrl": "https://datacatalog.worldbank.org/dataset/world-development-indicators/",
+      "observationPeriod": "P1Y"
+    },
+    "1151455814": {
+      "importName": "OECDRegionalDemography",
+      "provenanceUrl": "https://stats.oecd.org/Index.aspx?DataSetCode=REGION_DEMOGR#",
+      "measurementMethod": "OECDRegionalStatistics",
+      "observationPeriod": "P1Y"
+    },
+    "4181918134": {
+      "importName": "OECDRegionalDemography_Population",
+      "provenanceUrl": "https://data-explorer.oecd.org/vis?fs[0]=Topic%2C0%7CRegional%252C%20rural%20and%20urban%20development%23GEO%23&pg=40&fc=Topic&bp=true&snb=117&df[ds]=dsDisseminateFinalDMZ&df[id]=DSD_REG_DEMO%40DF_POP_5Y&df[ag]=OECD.CFE.EDS&df[vs]=2.0&dq=A.......&to[TIME_PERIOD]=false&vw=tb&pd=%2C",
+      "measurementMethod": "OECDRegionalStatistics",
+      "observationPeriod": "P1Y"
+    },
+    "1216205004": {
+      "importName": "CanadaStatistics",
+      "provenanceUrl": "https://www150.statcan.gc.ca/n1/en/type/data?MM=1"
+    }
+  }
+}
+```
+{: .example-box-content .scroll}
+
+{: .no_toc}
+#### Example 2: Get the latest observations for a single entity, filtering by provenance
+
+In this example, we again get the latest observations for `Count_Person`, but this time for the U.S., filtering for a single source, namely the U.S. government census, represented by its domain name, `www2.census.gov`.
+
+Request:
+{: .example-box-title}
+
+```python
+client.observation.fetch_observations_by_entity_dcid(date="latest", entity_dcids="country/USA", variable_dcids="Count_Person", filter_facet_domains="www2.census.gov")
+```
+{: .example-box-content .scroll}
+
+> Tip: This example is the equivalent of `client.observation.fetch(variable_dcids="Count_Person", entity_dcids="country/USA", filter_facet_domains="www2.census.gov")`.
```

Comment:
love these Tips, very helpful in understanding the overlap between the methods!!